src/libs/Navigation/AppNavigator/fullscreenModalcardStyleInterpolator.js

```diff
@@ -0,0 +1,42 @@
+import {Animated} from 'react-native';
+import variables from '../../../styles/variables';
+
+export default (
```

Comment:
Not too sure why we need the custom interpolator for this? It seems similar to the one we have already, but having trouble picking out the difference.

Reply:
I thought that sliding animation is not appropriate for the Web as Modal is fullscreen So I added fade on the web and used the existing slide on Mobile. I can combine that into a single file and probably share logic between them.